airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py

```diff
@@ -229,7 +239,7 @@ def _verify_read_output(output: Dict[str, Any], scenario: TestScenario) -> None:
         if "record" in actual:
             for key, value in actual["record"]["data"].items():
                 if isinstance(value, float):
-                    assert math.isclose(value, expected["data"][key], abs_tol=1e-06)
+                    assert math.isclose(value, expected["data"][key], abs_tol=1e-04)
```

Comment:
This one was interesting. There are some rounding errors when we try to write to Avro files in the test. I don't believe this is an issue reading, but rather when we have a Python float, we have to write it to an Avro value when creating the file. The resulting value read in has around 4ish trailing "close" values. So I loosened the accuracy a bit vs what I saw in my tests

https://stackoverflow.com/questions/45827782/python-fastavro-incorrectly-converting-float